pom.xml

```diff
@@ -497,14 +506,12 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
-    <raml-module-builder.version>30.2.7</raml-module-builder.version>
+    <raml-module-builder.version>31.1.0</raml-module-builder.version>
     <generate_routing_context>/users</generate_routing_context>
     <junit.version>5.5.2</junit.version>
     <rest-assured.version>3.3.0</rest-assured.version>
-    <folio-service-tools.version>1.5.1</folio-service-tools.version>
-    <mod-custom-fields.version>1.4.2-SNAPSHOT</mod-custom-fields.version>
+    <folio-service-tools.version>1.5.2-SNAPSHOT</folio-service-tools.version>
+    <folio-custom-fields.version>1.4.2-SNAPSHOT</folio-custom-fields.version>
```

Comment:
These really shouldn't be snapshot versions. Please could you follow up with the maintainers of these libraries to ask when these versions will be formally released?

Reply:
I have asked a question here https://issues.folio.org/browse/MODUSERS-215?focusedCommentId=87176&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-87176

Reply:
@marcjohnson-kint, here are the tickets to release those two utility modules:
* https://issues.folio.org/browse/FST-4
* https://issues.folio.org/browse/FCFIELDS-8